### DISCUSSION TITLE
Improvement of the lithium plating model

### DISCUSSION THREAD

**Discussion by xrfe** [pybamm-team/PyBaMM#3447] [Ideas]
From an experimental point of view, I believe that the current lithium-plating model is incomplete in describing the phenomenon of lithium plating at the anode. 

I have noticed that PyBaMM includes three different models: 1. irreversible, 2. partially reversible, and 3. reversible plating models. I suggest removing the irreversible implementation in PyBaMM because it contradicts experimental observations completely. The 'partially reversible' model is a suitable implementation, but I find the derivation of the equations somewhat ad hoc, making it difficult to discern the detailed physical interpretation of the decay coefficient.

Additionally, PyBaMM assumes that the plating reaction always occurs, while some researchers propose that plating happens when the surface overpotential is below zero. Both assumptions are plausible, but it seems that experiments lean towards the latter. However, neither of these approaches is sufficient to explain the onset of lithium plating. Perhaps the PyBaMM team could investigate lithium nucleation theory to create a better model for the lithium-plating process. Note that the nucleation process may always occur, but not necessary trigger the plating (continuous grwoth).

**Comment by brosaplanella** [comment]
@DrSOKane is our expert on plating so his input will be way more valuable than mine.

I would be against removing the irreversible one from PyBaMM. Even though it is not realistic, I think it does not hurt to have it as an option given that it is very common in the literature.

**Comment by DrSOKane** [comment]
Hi @xrfe! As the creator of PyBaMM's lithium plating model, I think this has the potential to be an interesting discussion. Let me start by explaining how I derived the equations.

The reversible model rests on three assumptions:
1. The transfer of an electron from graphite to Li+ (or vice-versa) is the rate-limiting step.
2. The equilibrium potential for the reaction is 0 V.
3. The reference concentration of Li metal is arbitrary and can therefore be set equal to that of Li+ in the electrolyte without any loss of generality.

Assumption 1 means the Butler-Volmer equation can be used. Assumption 2 requires a special case of the Butler-Volmer equation for which the concentration of reactants cannot be taken outside the brackets, as is the case for a finite equilibrium potential. Assumption 3 is not _required_ but relaxing it would introduce an additional fitting parameter, as demonstrated by [Yang et al.](https://doi.org/10.1016/j.jpowsour.2018.05.073).

The partially reversible model includes the reversible model in its entirety and therefore rests on the same assumptions. There is also one additional assumption:
4. The rate at which dead lithium is formed is directly proportional to the concentration of plated lithium.

Assumption 4 leads to the second, irreversible term on the right hand side of the differential equation for plating lithium concentration.

How valid are these assumptions? Assumptions 1 and 2 are standard in the literature. I agree with you that assumptions 3 and 4 are somewhat arbitrary, but I have yet to see a better alternative to either, nor am I aware of any experiments that directly contradict them, because lithium plating is very difficult to quantify.

**Comment by DrSOKane** [comment]
How to interpret the decay constant? I agree it's complicated. The best thing to do is start from the simplest case and add complexity gradually.

In the reversible model, it is possible to reach a state of thermodynamic equilribium in which the current for both the main reaction and the side reaction is zero. In this case, it is possible to rearrange the Butler-Volmer equation to find an equilibrium concentration of plating lithium, as a function of the open-circuit potential of the graphite.

In the partially reversible model, there is no thermodyanamic equilibrium, so consider the next simplest case, in which
a. The currents for the main reaction and side reaction sum to zero, and
b. The LiC6 concentration is such that the OCV is on a plateau and is therefore not going to change any time soon

In this special case, the solution to the PDE is that the (reversibly) plated lithium concentration never changes, because the reversible and ireversible terms on the right hand side add up to zero. Plated lithium turns into dead lithium at a rate determined by the constant plated lithium concentration and the decay constant. The plated lithium is replaced at an equal rate by plating from the electrolyte. To maintain charge neutrality in the electrolyte, the lithium must ultimately be taken from the graphite. This is where assumption b is massively helpful!

So, the rate constant can be interpreted a voltage-dependent calendar ageing rate similar to that widely used in SEI models. I appreciate this doesn't make it any easier to measure, but it does help with interpretation.

Another surprisingly simple case was discovered in [my own paper](https://doi.org/10.1039/d2cp00417h), in which the capacity fade during cycling ageing turns out to be almost proportional to the decay rate if the Butler-Volmer constant is very large. If the B-V constant is small, things get more complicated.
